src/main/java/studio/blockops/vyom/core/utils/Value.java

```diff
@@ -0,0 +1,60 @@
+package studio.blockops.vyom.core.utils;
+
+import java.util.Arrays;
+import java.util.List;
+
+// import studio.blockops.vyom.core.crypto.HashUtil;
+//import org.bouncycastle.util.encoders.Hex;
+//
+//import java.math.BigInteger;
+//
+//import java.util.Arrays;
+//import java.util.List;
+
+/**
+ * Class to encapsulate an object and provide utilities for conversion
+ */
+public class Value {
```

Comment:
Provide proper name to the class. Current name doesn't reflect it's purpose.

Reply:
I think it does